scripts/beesd.in

```diff
@@ -62,9 +56,13 @@ for arg in "${ARGUMENTS[@]}"; do
 done
 
 for arg in "${NOT_SUPPORTED_ARGS[@]}"; do
-    if uuid_valid $arg; then
-        [ ! -z "$UUID" ] && help
-        UUID=$arg
+    if [ ! -z "$(blkid --uuid $arg -o value | head -1)" ]; then
+        UUID="$arg"
+        INFO "Using UUID $arg"
+    fi
+    if [ ! -z "$(blkid /dev/mapper/$arg -o value | head -1)" ]; then
+        UUID="$(blkid /dev/mapper/$arg -o value | head -1)"
+        INFO "Found $UUID for $arg"
     fi
```

Comment:
This misses the "help" branch condition which has been there previously. Please convert to using `elif` and finally end with `help`.

Reply:
fair enough